<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Reload externally changed files:</string>
         </property>
         <property name="buddy">
          <cstring>reloadExternChangesCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="reloadExternChangesCombo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Always</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uiTab">
      <attribute name="title">
       <string>Interface</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Language:</string>
         </property>
         <property name="buddy">
          <cstring>languageCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="languageCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Theme:</string>
         </property>
         <property name="buddy">
          <cstring>themeCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QLabel" name="darkThemeLabel">
           <property name="text">
            <string>Automatically switch to the below theme if system dark mode is detected:</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>darkThemeCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="darkThemeCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="themeCombo"/>
         </item>
         <item row="1" column="0" rowspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>58</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Font size scale:</string>
         </property>
         <property name="buddy">
          <cstring>fontScaleCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="fontScaleCombo">
         <item>
          <property name="text">
           <string>100%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>125%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>150%</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gameTab">
      <attribute name="title">
       <string>Game</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Primary Minecraft version:</string>
         </property>
         <property name="buddy">
          <cstring>gameVersionCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="gameVersionCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Custom command tree file:</string>
         </property>
         <property name="buddy">
          <cstring>customCmdEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QToolButton" name="customCmdBtn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="customCmdEdit"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="customCmdStatus">
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="codeEditorTab">
      <attribute name="title">
       <string>Code editor</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Text size:</string>
         </property>
         <property name="buddy">
          <cstring>editorTextSizeSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="editorTextSizeSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>13</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Text font family:</string>
         </property>
         <property name="buddy">
          <cstring>editorTextFontCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFontComboBox" name="editorTextFontCombo">
         <property name="fontFilters">
          <set>QFontComboBox::MonospacedFonts</set>
         </property>
         <property name="currentFont">
          <font>
           <family>Courier New</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="editorWrapCheck">
         <property name="text">
          <string>Wrap text</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Block commenting mode:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QRadioButton" name="commentFillModeRadio">
           <property name="text">
            <string>Fill</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">commentMode</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="commentToggleModeRadio">
           <property name="text">
            <string>Toggle</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">commentMode</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Tab size:</string>
         </property>
         <property name="buddy">
          <cstring>editorTabSizeSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="editorTabSizeSpin">
         <property name="suffix">
          <string> spaces</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Custom code palette file:</string>
         </property>
         <property name="buddy">
          <cstring>customCodePaletEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="customCodePaletEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="customCodePaletBtn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="customCodePaletSampleBtn">
           <property name="text">
            <string>Get example...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="editorTabAsSpacesCheck">
         <property name="text">
          <string>Insert Tab characters as spaces</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="editorShowSpacesCheck">
         <property name="text">
          <string>Show spaces and tab characters</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>reloadExternChangesCombo</tabstop>
  <tabstop>languageCombo</tabstop>
  <tabstop>themeCombo</tabstop>
  <tabstop>darkThemeCombo</tabstop>
  <tabstop>gameVersionCombo</tabstop>
  <tabstop>customCmdEdit</tabstop>
  <tabstop>customCmdBtn</tabstop>
  <tabstop>editorTextSizeSpin</tabstop>
  <tabstop>editorTextFontCombo</tabstop>
  <tabstop>editorWrapCheck</tabstop>
  <tabstop>commentFillModeRadio</tabstop>
  <tabstop>commentToggleModeRadio</tabstop>
  <tabstop>editorTabSizeSpin</tabstop>
  <tabstop>editorTabAsSpacesCheck</tabstop>
  <tabstop>editorShowSpacesCheck</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="commentMode"/>
 </buttongroups>
</ui>
